src/Model/Worker.py

```diff
@@ -53,6 +54,8 @@ def __init__(self, fn, *args, **kwargs):
 
         if 'progress_callback' in self.kwargs and self.kwargs['progress_callback'] is True:
             self.kwargs['progress_callback'] = self.signals.progress
+        elif 'progress_callback' in self.kwargs and self.kwargs['progress_callback'] is False:
```

Comment:
elif 'progress_callback' in self.kwargs and not `self.kwargs['progress_callback']`
```suggestion
        elif 'progress_callback' in self.kwargs and not self.kwargs['progress_callback']:
```